### ISSUE TITLE
Warnings when set default_value

### ISSUE THREAD

**Issue by mrquokka** [sqlalchemy/sqlalchemy#4817]
```

engine = sqlalchemy.create_engine(
    'postgresql+psycopg2://' +
    postgresql_config['db_login'] + ':' +
    postgresql_config['db_password'] + '@' +
    postgresql_config['db_remote_adress'] + ':' +
    str(postgresql_config['db_remote_port']) + '/' +
    postgresql_config['db_name'],

    echo=postgresql_config['db_echo'],
    isolation_level='READ COMMITTED',
    pool_size=20, max_overflow=0)

engine.execute("SET TIME ZONE 0")
register_after_fork(engine, engine.dispose)
Base = sqlalchemy.ext.declarative.declarative_base()

class RootClass_(Base):
  __tablename__ = "rootclass_"
  mRID = sqlalchemy.Column(UUID, server_default=sqlalchemy.text(
      "uuid_generate_v4()"), primary_key=True)
  polymorphic_type = sqlalchemy.Column(sqlalchemy.types.String)
  __mapper_args__ = {
      'polymorphic_identity': __tablename__,
      'polymorphic_on': polymorphic_type
  }
RootClass_.cvs_marked_as_deleted = sqlalchemy.Column(
    sqlalchemy.types.Boolean, default_value=False)  
```
Error msg is 
```
/hdd/work/envs/main/lib/python3.7/site-packages/sqlalchemy/sql/base.py:299: SAWarning: Can't validate argument 'default_value'; can't locate any SQLAlchemy dialect named 'default'
    % (k, dialect_name)
```

**Comment by zzzeek** [comment]
OK and what would you expect to happen?

**Comment by zzzeek** [comment]
sorry I'll move past the snark.  There is no option "default_value", you are likely looking for "default". It does not raise a TypeError or other error because options can be added to any schema object of the form "<dialect>_<option>", such as "mysql_autoincrement".   The lookup of "<dialect>" is dynamic and it cannot assume the dialect is installed; it assumes you are naming a SQL dialect called "default".   So it can only warn.   The system was implemented in #2866.

**Comment by mrquokka** [comment]
> sorry I'll move past the snark. There is no option "default_value", you are likely looking for "default". It does not raise a TypeError or other error because options can be added to any schema object of the form `"<dialect>_<option>"`, such as "mysql_autoincrement". The lookup of `"<dialect>"` is dynamic and it cannot assume the dialect is installed; it assumes you are naming a SQL dialect called "default". So it can only warn. The system was implemented in #2866.

That's my fail, confused names (need use 'default' instead of 'default_value')
